plugins/vst_base/CMakeLists.txt

```diff
@@ -39,23 +39,43 @@ IF(LMMS_HOST_X86_64)
 	IF("${WINEBUILD_OUTPUT}" MATCHES ".*x86_64-linux-gnu/wine/libwinecrt0.a.*")
 		SET(EXTRA_FLAGS ${EXTRA_FLAGS} -nodefaultlibs /usr/lib/i386-linux-gnu/wine/libwinecrt0.a -luser32 -lkernel32 -lgdi32)
 	ENDIF()
-	#The following check works on Fedora systems 
-	IF("${WINEBUILD_OUTPUT}" MATCHES ".*lib64/wine/libwinecrt0.a.*")
+	# The following check works on Fedora systems
+	IF("${WINEBUILD_OUTPUT}" MATCHES "/usr/lib/lib64/wine/libwinecrt0.a.*")
 		SET(EXTRA_FLAGS ${EXTRA_FLAGS} -nodefaultlibs /usr/lib/i386/wine/libwinecrt0.a -luser32 -lkernel32 -lgdi32)
 	ENDIF()
+	# Development
+	IF("${WINEBUILD_OUTPUT}" MATCHES "/opt/wine-devel/lib64/wine/libwinecrt0.a.*")
```

Comment:
This workaround is currently here because some people do not know they have to install 32-bit Wine and do not understand the linker error. If you install the package from WineHQ, you have 64-bit and 32-bit Wine, so you cannot get this error because 32-bit is selected. Did you actually run into this problem?